src/hit/api/evaluator/homomorphic.cpp

```diff
@@ -308,4 +342,16 @@ namespace hit {
     void HomomorphicEval::relinearize_inplace_internal(CKKSCiphertext &ct) {
         relinearize(get_evaluator(), ct.backend_ct, relin_keys, ct.backend_ct);
     }
+
+    CKKSCiphertext HomomorphicEval::bootstrap_internal(const CKKSCiphertext &ct, bool rescale_for_bootstrapping) {
```

Comment:
Is it an internal method? I cannot find its usage.

Reply:
`bootstrap_internal` is the evaluator-specific bootstrapping function. The user API is here: https://github.com/awslabs/homomorphic-implementors-toolkit/blob/wip-bootstrapping-homomorphic/src/hit/api/evaluator.cpp#L382